core/capabilities/webapi/target/types.go

```diff
@@ -0,0 +1,19 @@
+package target
+
+const (
+	AllAtOnce  string = "AllAtOnce"
+	RoundRobin string = "RoundRobin"
+)
+
+type WorkflowInput struct {
```

Comment:
Not "WorkflowInput" - should be "WebAPITargetInput" or even simply "Input" since it's scoped by Go package name.